micro_dl/plotting/plot_utils.py

```diff
@@ -74,6 +76,98 @@ def save_predicted_images(input_batch, target_batch, pred_batch,
         plt.close(fig)
 
 
+def plot_xyz(image_dir,
+             pos_idx,
+             fig_name,
+             mean_std=None,
+             tol=1,
+             font_size=15,
+             color_map='gray',
+             fig_title=None,
+             margin=20,
+             z_scale=5,
+             channel_str=None):
```

Comment:
Please feel free to ignore this, it seems like we can group these params into image related and figure related params, may be two dicts image_params: {pos_idx, mean_std, tol, z_scale, channel_str} and fig_params:{fig_name, font_size, color_map, fig_title, margin}. Also could we please replace tol with per_clip_limits as tol reminds me of tolerance or epsilon

Reply:
tol was copied from the plot function above, so I'll update that one too while I'm at it. :-)

Reply:
yes please 😸